src/posts/lucky_0.23.cr

```diff
@@ -0,0 +1,219 @@
+    is out now and works with the newest version of Crystal (v0.35.0)
+
+    Be sure to upgrade your version of Crystal, and take a look at our
+    [UPGRADE NOTES](https://github.com/luckyframework/lucky/blob/master/UPGRADE_NOTES.md#upgrading-from-022-to-023)
+    for help with migrating your app.
+
+    ### LuckyDiff
+
+    Thanks to a community member [@stephendolan](https://github.com/stephendolan/lucky_diff) for
+    creating this awesome tool, we have started linking to it in our UPGRADE_NOTES to help you
+    see what the differences are between versions!
+
+    ### What happened to 0.22.0?
+
+    Don't worry, you didn't miss anything. If you're using Lucky 0.21.0, you can upgrade to
+    [0.22.0](https://github.com/luckyframework/lucky/blob/master/CHANGELOG.md#v0220-2020-06-17) just by
+    updating your Crystal version to 0.35.0. This release didn't change anything other than compatibility
+    with the latest Crystal. To get all of the latest goodies, you'll want to go up to 0.23.0.
+
+    ## Quick Updates
+
+    ### `mount` is now `m`
+
+    We have deprecated the use of `mount` in favor of a shorter method `m`. The structure of this
+    is a little bit different.
+
+    ```crystal
+    # Before
+    mount MyComponent.new(arg: 1)
+
+    # After
+    m MyComponent, arg: 1
+    ```
+
+    As you can see, you still have all the flexibility, but in a bit more compact version!
+
+    ### Params
+
+    When you need to access a param value, you'll use `params.get(:key)`, but now, the blankspace
+    will be stripped automatically. If you need to get the untouched value, you can use `params.get_raw`.
+
+    ### Memoize 2.0
+
+    The `Memoizable` module got a nice little upgrade. It now supports arguments, as well as
+    `false` and `nil` values. 
+
+    ```crystal
+    memoize user(user_id : Int64) : User?
+      UserQuery.new.find(user_id)
+    end
+    ```
+
+    Now when you call this method, it will run the first time, and return the value even if that value is
+    `nil`. If you need access to the uncached version you can use the `__uncached` version of the method.
+
+    ```crystal
+    # run the query once, and always return that value
+    user(4)
+
+    # always run the query and return the value
+    user__uncached(4)
+    ```
+
+    ### Security and more
+    
+    Some of the HTML page helpers like `highlight`, and `simple_format` would allow for unescaped HTML.
+    This could potentially be a security issue, so that's been fixed. All of these methods will escape the
+    HTML by default now. If you need the unescaped version, you can pass `escape: false`.
+
+    ```crystal
+    highlight("<p>This is a beautiful morning, but also a beautiful day</p>", "beautiful", escape: false)
+    ```
+
+    We've also fixed a few issues related to cookies.
+    * You can now set a `samesite` default.
+    * Exceptions aren't raised when trying to delete a cookie that doesn't exist
+    * Cookies are written at the correct time
```

Comment:
I'd leave this out just because I think it is super hard to explain what's going on and without the explanation it doesn't provide much value. What do you think?